app/components/drag-handle.js

```diff
@@ -1,5 +1,8 @@
 import Ember from "ember";
-const { computed, String: { htmlSafe }, K } = Ember;
+
+const { computed, String: { htmlSafe } } = Ember;
+const K = function() { return this; }
```

Comment:
IMHO, the replacement here is simply to use empty functions where needed. We don't need another "special" Emberism, let's just use functions.